### ISSUE TITLE
 Aboveandbeyond - Fake news detector - Social Impact

### ISSUE THREAD

**Issue by mayankchauhan96** [skillenza-com/MishMash-India-2020#170]
Aboveandbeyond - Fake news detector -  Social Impact

## ℹ️ Project information

 1. **Theme**:   Social Impact

2. **Project Name:** Fake news detector

3. **Short Project Description:** Technical solution to stop fake news spreading related to Covid 19 Epidemic 

4. **Team Name:**  Aboveandbeyond

5. **Team Members:** Raghav Toshniwal 

7. **Repository Link(s):**  https://github.com/mayankchauhan96/Fake-news-detector

## 🔥 Your Pitch

Use Case:

As the SARS-CoV-2 virus – more popularly known as COVID-19 – has spread around the globe, conspiracy theories and rumours have also gone viral on social media platforms and other outlets. Or we can rather state them as "infodemics", these can result in a very hazardous outcome if someone/community is believing them (The news which you all must have heard about the loss of lives caused by alcohol consumption by some citizens of Italy who were even found negative after their test).

How this works?

Basically we building awhatsapp app which will help in getting client's query and response back the correct information. In initial stage we have to volunteer some individuals which will mark the relevant flag against that news/post in front end web app. Later we are planning to automatically verify the media source with the help of ML models. This will also work for all other news topics as well.
